openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -153,7 +153,11 @@ <h4 class="subsection-title">
                     % for vertical in subsection.get('children', []):
                                     <li class="vertical outline-item focusable">
                                         <a class="outline-item focusable"
-                                            href="${ vertical['lms_web_url'] }"
+                                            % if enable_links:
+                                                href="${ vertical['lms_web_url'] }"
```

Comment:
I think the quality linter error is a real one, and points to not HTML escaping all the outputs in this Mako template.

Reply:
Okay, took me some time to figure this one out.  The three issues were:
```
Linting common/lib/xmodule/xmodule/x_module.py:
common/lib/xmodule/xmodule/x_module.py: 386:16: python-deprecated-display-name:         return block_metadata_utils.display_name_with_default_escaped(self)
common/lib/xmodule/xmodule/x_module.py: 491:21: python-deprecated-display-name:             return [self.display_name_with_default_escaped]
common/lib/xmodule/xmodule/x_module.py: 873:17: python-wrap-html:         return u'<x_module(id={0})>'.format(self.id)
```

1. I think the xsscommitlint checks any file which has been edited in the commit and not just sections of the code which have been modified. Verified this by undoing the changes in `x_module.py`. You can see the test run for the 2nd commit does not show these issues.
2. Having said so all 3 are false positives as far as I have been able to figure so I have disabled them in the 4th commit. But let me know if I am missing something here.

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#python-deprecated-display-name
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#python-wrap-html

Reply:
I can re-squash if necessary.

Reply:
Got it. Yes, please re-squash, and thank you for working that out.